client/client.go

```diff
@@ -186,16 +190,8 @@ func (ec *SDKClient) Balance(
 
 	blockNum := hexutil.EncodeUint64(head.Number.Uint64())
 	reqs := []rpc.BatchElem{
-		{
-			Method: "eth_getBalance",
-			Args:   []interface{}{account.Address, blockNum},
-			Result: &nativeBalance,
-		},
-		{
-			Method: "eth_getTransactionCount",
-			Args:   []interface{}{account.Address, blockNum},
-			Result: &nonce,
-		},
+		{Method: "eth_getBalance", Args: []interface{}{account.Address, blockNum}, Result: &nativeBalance},
```

Comment:
isn't the earlier formatting more readable? how did this formatting change happen?